ms-engine/src/main/java/net/trajano/ms/engine/SpringJaxRsHandler.java

```diff
@@ -138,7 +143,9 @@
      * @param applicationClass
      *            application class
      * @return the handler
+     * @deprecated use {@link JaxRsRouter}
      */
+    @Deprecated
     public static SpringJaxRsHandler registerToRouter(final Router router,
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)